contest/contest_info.py

```diff
@@ -63,10 +63,38 @@ def get_running_contests():
 def get_contestant_list(contest):
     return Contestant.objects.filter(contest = contest)
 
+
+def get_contestant(contest):
+    contestants = Contestant.objects.filter(contest=contest)
+    return [contestant.user for contestant in contestants]
+
+
 def get_total_testcases(problem):
     testcases = Testcase.objects.filter(problem = problem)
     return testcases.count()
 
+
+def get_contest_submissions(contest, submissions):
```

Comment:
Can I simplify my design that is to freeze all submissions after the freeze time in the status? Then, it can be searched no matter what.